CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # nf-core/tools: Changelog
 
+## v1.3dev
```

Comment:
```suggestion
## v1.4dev
```

Reply:
Sorry. Thats exactly what I meant!

Reply:
How do you make these changes after merging @apeltzer? My naive brain is trying to work it out!

Reply:
My bad - I've merged before really thinking as I thought the commit was fine and good (expect for these smallish typos). I guess just make a new PR with just these two then and I'll merge it in... sorry about that. 

Normally you can simply click "accept" and thats it... :-)

Reply:
No worries. I have fixed this in https://github.com/nf-core/tools/pull/214 whilst I was dealing with another issue.